client/src/components/ImageList.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react'
-import styled from 'styled-components'
+//import styled from 'styled-components'
 
-const ImageWrapper = styled.div`
+const ImageWrapper = window.styled.div`
```

Comment:
how did you figure out that this gets rid of the styled-component errors?

Reply:
i install it on a cdn on the html page, which makes it available on the window object, 
but its not good because its cluttering up the global namespace